template/.gitlab-ci.yml

```diff
@@ -1,8 +1,36 @@
-include:
-    - project: Ackee/gitlab-ci-templates
-      ref: nextjs-cloudrun/v1.0.1
-      file: nextjs-cloudrun/pipeline.yml
+default:
+  image: node:16.16.0
 
 variables:
-    NODE_BASE_IMAGE: 'node:16.19'
-    DEPLOYMENT_TYPE: 'cloudrun'
+  ACKEE_VERCEL_API_TOKEN: ""
```

Comment:
This should rather be in vault, not in a project, but I guess we can fine-tune the pipeline later.